recipes/metamorpheus/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eox pipefail
+
+PREFIX=$(echo "${PREFIX}" | tr '\\' '/')
+
+if [[ "${build_platform}" == "win-64" ]]; then
+    DOTNET_ROOT="${PREFIX}/dotnet"
+else
+    DOTNET_ROOT="${PREFIX}/lib/dotnet"
+fi
+
+METAMORPHEUS_ROOT=$DOTNET_ROOT/tools/metamorpheus
+
+mkdir -p $PREFIX/bin $METAMORPHEUS_ROOT
+cp -r $SRC_DIR/* $METAMORPHEUS_ROOT
+
+if [[ "${build_platform}" == "win-64" ]]; then
```

Comment:
I guess you actually care about the target platform?
```suggestion
if [[ "${target_platform}" == "win-64" ]]; then
```